src/js/components/Menu/__tests__/__snapshots__/Menu-test.js.snap

```diff
@@ -605,6 +602,7 @@ exports[`Menu close by clicking outside 2`] = `
       <button
         aria-label="Close Menu"
         class="c5"
+        tabindex="-1"
```

Comment:
The addition of these lines in the snapshots refers to the change in line 209 of Menu.js. Essentially, when the menu first opens, the user's tab should go to the first menu item not the control button. What this means is at first, the control button receives a tabindex of -1 to remove it from the tab order (otherwise it naturally receives focus because it's a button and if it is on top of the menu, it will take the focus first). However, once the user has tabbed once into the menu, the control button tabindex of -1 is removed.

Reply:
Thank you for anticipating my next question...